ledger-service/http-json/src/failure/scala/http/FailureTests.scala

```diff
@@ -96,40 +96,48 @@ final class FailureTests
     import json.JsonProtocol._
     for {
       p <- allocateParty(client, "Alice")
-      (status, _) <- postCreateCommand(
-        accountCreateCommand(p, "23"),
-        encoder,
-        uri,
-        headersWithParties(List(p.unwrap)),
+      (status, _) <- headersWithParties(List(p.unwrap)).flatMap(
+        postCreateCommand(
+          accountCreateCommand(p, "23"),
+          encoder,
+          uri,
+          _,
+        )
       )
       _ = status shouldBe StatusCodes.OK
       // Client -> Server connection
       _ = proxy.toxics().timeout("timeout", ToxicDirection.UPSTREAM, 0)
       body <- FutureUtil.toFuture(
         encoder.encodeCreateCommand(accountCreateCommand(p, "24"))
       ): Future[JsValue]
-      (status, output) <- postJsonStringRequestEncoded(
-        uri.withPath(Uri.Path("/v1/create")),
-        body.compactPrint,
-        headersWithParties(List(p.unwrap)),
+      (status, output) <- headersWithParties(List(p.unwrap)).flatMap(
+        postJsonStringRequestEncoded(
+          uri.withPath(Uri.Path("/v1/create")),
+          body.compactPrint,
+          _,
+        )
```

Comment:
This is an alternative that might work, but only if you think it is a more obvious and reviewable refactoring.

```suggestion
      (status, output) <- postJsonStringRequestEncoded(
          uri.withPath(Uri.Path("/v1/create")),
          body.compactPrint,
          _,
        ) =<< headersWithParties(List(p.unwrap))
```

Reply:
The flatMap operator might do the job here too, at least to get rid of all the braces.

Reply:
decided to not go with operators, it didn't make much of difference :(

Reply:
I was really thinking about the diff, if headersWithParties is closer to where it was it might be easier to read. But, again, whatever works out best in practice 🤷